### PR TITLE
[ckcore] set host to 0.0.0.0 inside docker by default

### LINKED ISSUE #476: [docker] bind to 0.0.0.0 by default when running inside docker

### PR DESCRIPTION
**Pull request by aquamatthias**
Fixes #476

### CODE REVIEW

**Comment by lloesche** [comment]
We should do this within the `docker/defaults` file instead of inside `ckcore`.

**Review by lloesche — CHANGES_REQUESTED** [review]
Exporting `CKCORE_HOST=0.0.0.0` via `docker/defaults` and inside the `bootstrap` script should be enough to achieve this. No change in `ckcore` itself needed.

File: ckcore/core/__main__.py [hunk 1]
```diff
@@ -109,7 +109,7 @@ async def on_start() -> None:
                 "created_at": to_json(system_data.created_at),
                 "system": platform.system(),
                 "platform": platform.platform(),
-                "inside_docker": in_docker,
+                "inside_docker": inside_docker,
```
Comment (lloesche) [inline ckcore/core/__main__.py:112]: What is this being used for and how does it relate to binding on any interface?

**Review by lloesche — CHANGES_REQUESTED** [review]

File: ckcore/core/__main__.py (hunk 1 above)
Comment (aquamatthias) [reply ckcore/core/__main__.py:112]: somehow unrelated - the variable existed. I just renamed it. Does not change any logic.

**Pull request merged**